Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1817,9 +1829,10 @@ namespace Remotion.BocAutoCompleteReferenceValue
         const referenceWidth = LayoutUtility.GetOuterWidth(reference);
         const marginLeft = 30;
         const availableWidth = position.left + referenceWidth - marginLeft;
+        const requireScrollbar = requiredHeight >= maxHeightSafe;
```

Comment:
```suggestion
        const isScrollbarRequired = requiredHeight >= maxHeightSafe;
```

booleans should follow the `is...` or `has...` naming convention whenever reasonably possible.